apps/shipments/serializers/shipment_action.py

```diff
@@ -36,6 +36,7 @@ class ShipmentActionRequestSerializer(serializers.Serializer):
     tracking_data = serializers.CharField(required=False, allow_null=True)
     document_id = serializers.CharField(required=False, allow_null=True)
     raw_asset_physical_id = serializers.CharField(required=False, allow_null=True)
+    asset_physical_id = serializers.CharField(required=False, allow_null=True)
```

Comment:
I think in `validate_action_type` we should also validate that the requester has the proper gtx permission.
Also the way it's set up now, only shipments with `gtx_required = True` will be able to use this feature. I think we could be a bit flexible on that in this method by allowing any user with gtx permission to be able to perform the pick up action.

Reply:
The current implementation conforms to the requirements.

The pick up action is not concerned with the requester having `gtx.shipment_use` permission.  That permission is needed for defining that a shipment _requires_ the GTX flow (i.e. setting `gtx_required = True`).  Once a shipment has that flag set to True, then the pickup action requires that an `asset_physical_id` be associated at pickup time and we no longer care about the requesting user's permission for enabling GTX.